src/runtime/string.go

```diff
@@ -32,6 +32,11 @@ func stringEqual(x, y string) bool {
 	return true
 }
 
+var (
+	// Constants for use with alloc()
+	gcLayoutNoPtrs = unsafe.Pointer(uintptr(0x3))
+)
```

Comment:
Perhaps we should move constants like these to a separate internal package? This is quite finicky stuff and easy to get wrong.